src/components/MeasurementSelect/MeasurementSelect.tsx

```diff
@@ -0,0 +1,33 @@
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import * as classes from './styles';
+
+interface Props {
+  unit: string;
+  setUnit: (value: string) => void;
```

Comment:
make `unit` and `value` stricter. It accepts not any string by a fixed range of values